frontend/src/lib/env.js

```diff
@@ -2,12 +2,14 @@
  * Environment variables
  */
 
-if (typeof process.env.NODE_ENV === 'undefined') {
-  process.env.NODE_ENV = 'development';
+let NODE_ENV = process.env.NODE_ENV;
+
+if (NODE_ENV === 'undefined') {
+  NODE_ENV = 'development';
```

Comment:
You can't set `process.env.NODE_ENV`, it throws an error and the entire application grinds to halt…